test/test_models.py

```diff
@@ -23,6 +25,36 @@ def get_models_from_module(module):
     return [v for k, v in module.__dict__.items() if callable(v) and k[0].lower() == k[0] and k[0] != "_"]
 
 
+@pytest.fixture
+def disable_weight_loading(mocker):
```

Comment:
Thanks @pmeier 

Considering this is a fairly complex logic, it might be useful to future us to add a few comments about what this fixture is doing, and most importantly *why* it's doing it the way it's doing it.

Reply:
Thank you, great docs :)